x/bank/keeper/keeper.go

```diff
@@ -173,6 +173,13 @@ func (k BaseKeeper) DelegateCoins(ctx sdk.Context, delegatorAddr, moduleAccAddr
 		return sdkerrors.Wrapf(sdkerrors.ErrUnknownAddress, "module account %s does not exist", moduleAccAddr)
 	}
 
+	// do not allow delegation if clawback vesting account
+	acc := k.ak.GetAccount(ctx, delegatorAddr)
+	_, ok := acc.(vestexported.ClawbackVestingAccountI)
+	if ok {
```

Comment:
So we are saying that a clawback vesting account can never delegate? I thought it was only the locked tokens that couldn't. 

Not really a big deal, since upon vest the recipient can send to a new address, and we can resolve this later. I think we should file an issue for this, but its not at all a priority to resolve.

Reply:
yeah that was my understanding as well(to restrict vesting accounts from staking). If not, we can definitely file an issue and revisit